packages/vue/src/index.js

```diff
@@ -40,10 +40,10 @@ export {
 	MultiList,
 	SingleRange,
 	ReactiveComponent,
-	SelectedFilters,
+	SelectedFilters
```

Comment:
trailing commas missing. 

Please make sure your linting rules are in sync with other packages.

Reply:
I think it's because you're using `aibnb` plugin & I'm using `airbnb-base` plugin which works with vue.
I can add extra rules for these configurations.